### PR TITLE
WIP: Add grinder and options to test_personal

### PR DESCRIPTION
**Pull request by smlambert**
- ITERATIONS is the number of times to execute a test target
- EXTRA_OPTIONS appends commandline options to the target being run
- JVM_OPTIONS is supposed to wholesale replace the commandline options being run

### CODE REVIEW

**Comment by smlambert** [comment]
I will be changing the playlists shortly to have a < test > like that in jck/playlist.xml (jck-runtime-custom) to target individual test classes for personal test builds.

File: buildenv/jenkins/Jenkinsfile_personal [hunk 1]
```diff
@@ -70,10 +76,14 @@ pipeline {
...
 							DIAGNOSTICLEVEL ='noDetails'
 						}
-					}
-					withEnv(["DIAGNOSTICLEVEL=${DIAGNOSTICLEVEL}"]) {
-						sh 'printenv'
-                		sh "$OPENJDK_TEST/maketest.sh $OPENJDK_TEST _${params.TARGET}"
+					
+						withEnv(["DIAGNOSTICLEVEL=${DIAGNOSTICLEVEL}"]) {
+							sh 'printenv'
+							int iterations = "${ITERATIONS}".toInteger()
```
Comment (sophia-guo) [inline buildenv/jenkins/Jenkinsfile_personal]: Not sure if the following script  needs  to be wrapped in script {}.

Comment (smlambert) [reply buildenv/jenkins/Jenkinsfile_personal]: A previous thing I tried did need it, I will try now without it to see if it works without.  Thanks!

Comment (smlambert) [reply buildenv/jenkins/Jenkinsfile_personal]: Need to keep the .toInteger() inside the block

**Comment by smlambert** [comment]
I would additionally like to re-add the NODE_LABEL field (so that someone could pass in a particular node name, if they wanted to grind on a machine of their choice), but may do that in a separate pull request.

File: openjdk_regression/playlist.xml [hunk 2]
```diff
@@ -14,6 +14,27 @@
 -->
 <playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../TestConfig/playlist.xsd">
 	<include>openjdk_regression.mk</include>
+	<test>
+		<testCaseName>openjdk-custom</testCaseName>
```
Comment (sophia-guo) [inline openjdk_regression/playlist.xml]: Better use same name converntions as other testcases: jdk_custom. As we map testcasename or target to build_list.

Comment (smlambert) [reply openjdk_regression/playlist.xml]: Yes, I was debating on that, or on making a standard naming that would be applied at each of the groups...  to match the jck-custom, openjdk-custom, perf-custom, external-custom, functional-custom... but those other types are difficult to customize, so I will use jdk_custom for now.

**Comment by smlambert** [comment]
Replaced by #412

**Pull request closed without merging**